nncf/experimental/openvino_native/graph/model_transformer.py

```diff
@@ -312,6 +317,6 @@ def _apply_model_extraction_transformation(model: ov.Model,
                 results.append(new_result)
 
         if not results:
-            results = [r.node for r in model.outputs]
+            results = model.get_results()
```

Comment:
Important. For the BC algorithm, it's needed to also check out the model creating with the **Assign** layers. But the model is affected by problems from 104117 and I suggest postponing this for the future.
cc @l-bat, @alexsu52